about.html

```diff
@@ -0,0 +1,170 @@
+
+        </section>
+
+ <!-- PAST -->
+
+        <section class="flex justify-center items-center flex-col py-20 md:px-20 border-b-2 ">
+
+            <h2 class=" hover:text-[#f28f10] text-2xl md:text-4xl font-semibold">
+                SEE PAST CONFERENCES 
+            </h2>
+
+            <div class="mp-underline"></div>
+
+            <p class="my-20 px-4">Take a look at the recent Conferences.</p>
+
+            <div class="flex flex-col gap-[4vw] md:flex-row">
+                <div class="h-[170px] w-[270px] md:h-[200px] md:w-[300px] lg:h-[300px] lg:w-[500px] flex justify-center items-center flex-col audi1">
+                    <div class="bg-[#f290108d] hover:bg-black/50 h-full w-full text-center flex justify-center items-center flex-col">
+                        <h4 class="text-[white] text-[2rem] shadow-black shadow-2xl">2020</h4>
+                        <p class="text-[white] shadow-black shadow-2xl">Travel Fest 2020</p> 
+                    </div>
+                </div>
+                <div class="h-[170px] w-[270px] md:h-[200px] md:w-[300px] lg:h-[300px] lg:w-[500px]  flex justify-center items-center flex-col audi2">
+                    <div class="bg-[#f290108d] hover:bg-black/50 h-full w-full text-center flex justify-center items-center flex-col">
+                        <h4 class="text-[white] text-[2rem] shadow-black shadow-2xl">2021</h4>
+                        <p class="text-[white] shadow-black shadow-2xl">Travel Fest 2021</p> 
+                    </div>
+                </div>
+            </div>
+
+        </section>
+
+<!-- PARTNERS -->
+
+<section class="bg-[#272a31] text-white flex justify-center items-center flex-col py-10">
+    <h2 class="mp-txt text-2xl md:text-4xl font-semibold hover:text-[#f28f10]">
+        PARTNERS
+    </h2>
+
+    <div class="mp-underline"></div>
+
+    <div class="mt-14 w-full flex justify-center items-center gap-14 flex-col md:flex-row">
+
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/bmw-logo-svgrepo-com.svg" alt="bmw logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/apple-logo-svgrepo-com.svg" alt="apple logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/google-icon-logo-svgrepo-com.svg" alt="google logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/instagram-2-1-logo-svgrepo-com.svg" alt="insta logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/whatsapp-icon-logo-svgrepo-com.svg" alt="wa logo">
+
+    </div>
+
+</section>
+
+    </main>
+
+<!-- FOOTER SECTION -->
+
+<footer class="flex w-full justify-center p-2 gap-2 items-center my-8">
+    <div class="flex justify-end w-1/2">
+    <img class="h-20 object-contain" src="./Assets/partners/CreativeCommons_logo_trademark.svg" alt="cc logo">
+    </div>
+    <div class="w-1/2">
+    <h2 class="hover:text-[#f28f10]">© All RIGHTS RESERVED 2022</h2>
+    </div>
+</footer>
```

Comment:
The footer section on the about page should be different from the main page. Kindly change the background color and please add the missing text to make your project stick as much as possible to the template provided. :+1: :smiley: :smiley: 
Your design
![image](https://user-images.githubusercontent.com/98363075/203322629-39acdf8c-7b70-491e-833e-f4dbd9287b8e.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203322723-eadec59d-013e-42bd-a35b-c9f9eb7e41ac.png)